### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [MahApps/MahApps.Metro#4291] [General]
Any time frame for the release of v3? 
I'm looking forward to use the new controls.

**Comment by punker76** [comment]
I'm currently working on it. Will finish it soon.

**Reply by KRAUS1203** [reply]
@punker76 
Hi, do you have any timeline when you will release v3?

**Reply by IvanSadovskyi** [reply]
It's been a year now... Is the project dead?

**Reply by fredericoregateiro** [reply]
The [3.0.0-alpha0457](https://www.nuget.org/packages/MahApps.Metro/3.0.0-alpha0457) was released 2 months ago.
So it's in the oven 🤣

**Reply by timunie** [reply]
Life came and has mage the development slowing down (a bit). But it's still being worked on and is still planned to get out v3.

**Reply by punker76** [reply]
I will do my best!

**Comment by makingbloke** [comment]
For which (and hopefully everyone else won’t mind me speaking on their
behalf) we are extremely grateful.

You guys do a brilliant job. MahApps.Metro is a joy to use and a lot more
reliable and better supported than many commercial libraries. It’s a
testament to all your knowledge and skills.

I’m sure if there was a way to send beer you’d never need to buy a pint
again :)

Cheers

**Comment by punker76** [comment]
I'm waiting now for SignPath to sign the NuGet package and dlls.
